itest/starboard_cli_test.go

```diff
@@ -710,8 +696,143 @@ var _ = Describe("Starboard CLI", func() {
 
 	})
 
-	PDescribe("Command audit configuration", func() {
-		// TODO Test polaris command
+	Describe("Command polaris", func() {
+		// containerNameAsIDFn is used as an identifier by the MatchAllElements matcher
+		// to group ConfigAuditReport by container name.
+		resourceNameAsIDFn := func(element interface{}) string {
+			return element.(v1alpha1.ConfigAuditReport).
+				Labels[kube.LabelResourceName]
+		}
+
+		FContext("when unmanaged Pod is specified as workload", func() {
```

Comment:
Focused spec or context is only for local dev. We should not commit such change to Git. It's also failing the build with the following message:

Detected Programmatic Focus - setting exit status to 197

```suggestion
		Context("when unmanaged Pod is specified as workload", func() {
```